### PR TITLE
(1211) Fix - Add back buttons

### PR DESCRIPTION
**Pull request by richpjames**
Adds back buttons where they were previously missing on: 
- Select offence screen
- Exceptional case screen
- Placement purpose screen
- Rehabilitative interventions screen

### CODE REVIEW

File: server/testutils/factories/application.ts [hunk 1]
```diff
@@ -33,6 +33,16 @@ class ApplicationFactory extends Factory<ApprovedPremisesApplication> {
       },
     })
   }
+
+  withPageResponse(section: string, task: string, key: string, value: unknown) {
```
Comment (pezholio) [inline server/testutils/factories/application.ts:37]: ❤️ Love it!

Comment (pezholio) [reply server/testutils/factories/application.ts:37]: Wonder if named parameters would make this clearer. Something like:

```
withPageResponse(args: {section: string, task: string, key: string, value: unknown})
```

**Review by pezholio — APPROVED** [review]
LGTM - just a suggestion for that factory helper

File: server/testutils/factories/application.ts (hunk 1 above)
Comment (richpjames) [reply server/testutils/factories/application.ts:37]: Yeah good point!

**Pull request merged**